### PR TITLE
Fix search index for incremental update

### LINKED ISSUE #5031: "pod search ReactiveCocoa" fails
# Report

## What did you do?
'''pod search ReactiveCocoa'''
## What did you expected to happen?

Find the ReactiveCocoa pod spec information.

## What happened instead?

Lots of other pods are listed that have that name as part of the string.  Except the ReactiveCocoa pod.   Which is weird.  Works on the website search.  But not the command line.

## Podfile

N/A  Not related to a pod file!

### PR DESCRIPTION
**Pull request by manuyavuz**
Fixes #5031.

I think we should provide an easy way to force reindexing for possible future changes on search index structure or to fix this type of mistakes. Any suggestions?

### CODE REVIEW

File: lib/cocoapods/sources_manager.rb [hunk 1]
```diff
@@ -279,15 +279,17 @@ def update_search_index_if_needed(changed_spec_paths)
 
           new_index = aggregate.generate_search_index_for_changes_in_source(source, spec_paths)
           # First traverse search_index and update existing words
-          # Removed traversed words from new_index after adding to search_index,
+          # Remove traversed words from new_index after adding to search_index,
```
Comment (orta) [inline lib/cocoapods/sources_manager.rb]: removes :)

Comment (manuyavuz) [inline lib/cocoapods/sources_manager.rb]: I was using the imperative form :)

**Comment by segiddins** [comment]
You need to run `bundle install` on your last commit, @manuyavuz

**Comment by segiddins** [comment]
This should be :+1: with a CHANGELOG entry, @manuyavuz

**Comment by segiddins** [comment]
These changes are now in https://github.com/CocoaPods/Core/pull/315.

**Pull request closed without merging**